ClosedXML/Excel/RichText/XLFormattedText.cs

```diff
@@ -169,7 +169,7 @@ public bool Equals(IXLFormattedText<T> other)
 
             for (Int32 i = 0; i < count; i++)
             {
-                if (_richTexts.ElementAt(i) != other.ElementAt(i))
+                if (_richTexts.ElementAt(i).Equals(other.ElementAt(i)))
```

Comment:
Should the `==` and `!=` operators not be overloaded to avoid this situation?

Reply:
Agree, it's a good practice to have a consistent behavior between Equals and ==

Reply:
Possible NRE.